stl/inc/ranges

```diff
@@ -22,14 +25,409 @@ _STL_DISABLE_CLANG_WARNINGS
 
 _STD_BEGIN
 namespace ranges {
-    // Much machinery defined in <xutility>
+    // MUCH machinery defined in <xutility>
 
     // clang-format off
     // CONCEPT ranges::viewable_range
     template <class _Rng>
     concept viewable_range = range<_Rng>
         && (borrowed_range<_Rng> || view<remove_cvref_t<_Rng>>);
 
+    template <class _Ty>
+    concept _Copy_constructible_object = copy_constructible<_Ty> && is_object_v<_Ty>;
+    // clang-format on
+
+    // CLASS TEMPLATE ranges::_Semiregular_box
+#if 0 // TRANSITION, VSO-1174090

```

Comment:
Does this bug affect Clang as well? Why not turn on the non-workaround for Clang? No change requested.

Reply:
Stephan made the same comment: https://github.com/microsoft/STL/pull/1201#discussion_r473453865. TLDR: The two alternatives are link-incompatible, and we support linking Clang-compiled and MSVC-compiled TUs.

Reply:
I think it's reasonable to keep if we expect to activate it soon.